src/main/kotlin/lotto/model/LottoResult.kt

```diff
@@ -0,0 +1,13 @@
+package lotto.model
+
+class LottoResult(private val result: Map<Rank, Int>) {
+    fun getTotalAmount(): Int {
+        var totalAmount = 0
+        result.forEach { (rank, count) -> totalAmount += rank.winningAmount * count }
+        return totalAmount
+    }
```

Comment:
```suggestion
    fun getTotalAmount(): Int {
        return result.entries.fold(0) { acc, (rank, count) -> acc + rank.winningAmount * count }
    }
```
kotlin Collection 의 fold 함수를 활용해볼 수도 있겠습니다~